backend/compact-connect/lambdas/python/purchases/tests/unit/test_purchase_client.py

```
@@ -274,7 +319,7 @@ def test_purchase_client_makes_successful_transaction_using_authorize_net_proces
         self.assertEqual('testFirstName', api_contract_v1_obj.transactionRequest.billTo.firstName)
         self.assertEqual('testLastName', api_contract_v1_obj.transactionRequest.billTo.lastName)
         # transaction settings
-        self.assertEqual('180', api_contract_v1_obj.transactionRequest.transactionSettings.setting[0].settingValue)
+        self.assertEqual('35', api_contract_v1_obj.transactionRequest.transactionSettings.setting[0].settingValue)
```

Pylint (3.3.7)

[convention] Line too long (114/100)

(C0301)